src/umpire/strategy/DynamicPool.hpp

```diff
@@ -50,13 +65,15 @@ class DynamicPool :
      * \param min_initial_alloc_size The minimum size of the first allocation
      *                               the pool will make.
      * \param min_alloc_size The minimum size of all future allocations.
+     * \param h_fun Heuristic callback function.
      */
     DynamicPool(
         const std::string& name,
-        int id,
+        int _id,
```

Comment:
This name change isn't required.

```suggestion
        int id,
```